server/Makefile

```diff
@@ -48,7 +48,6 @@ endif
 
 ifeq ($(BUILD_NUMBER),dev)
 	export MM_FEATUREFLAGS_GRAPHQL = true
-	GOTAGS += "testlicensekey"
```

Comment:
Should we automatically set the external environment env var here to `Dev` so it's defaulted for local dev?

Reply:
I can certainly do that, but I found that wasn't sufficient, since a non-Makefile invocation like `go test` wouldn't automatically get the setting applied. In `model.GetExternalServiceEnvironment`, I'm detecting the presence of a non-dev build number to try to assert with confidence that we're allowed to use the production external environment.